app/subscriber/src/components/basic-search/styled/BasicSearch.tsx

```diff
@@ -64,7 +63,7 @@ export const BasicSearch = styled(Row)<{ inHeader?: boolean }>`
       height: 2em;
     }
 
-    width: 85%;
+    width: calc(90% - 15px);
```

Comment:
15px is curiously close to the value of 1rem. Is there a 1rem thing you are decreasing the width for here?

Reply:
switched to 1rem